ntoskrnl/io/pnpmgr/pnpmgr.c

```diff
@@ -1853,7 +1853,7 @@ IopGetParentIdPrefix(PDEVICE_NODE DeviceNode,
     }
 
     /* 1. Try to retrieve ParentIdPrefix from registry */
-    KeyNameBufferLength = FIELD_OFFSET(KEY_VALUE_PARTIAL_INFORMATION, Data[0]) + MAX_PATH * sizeof(WCHAR);
+    KeyNameBufferLength = FIELD_OFFSET(KEY_VALUE_PARTIAL_INFORMATION, Data[0]) + 32 * sizeof(WCHAR);
```

Comment:
What does this "32" value corresponds to? some sizeof() ? some length of a string?

Reply:
Will change to `sizeof(L"12345678&12345678")`, which is what this function writes in there... nobody else should write these anyway.